app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -89,6 +89,18 @@ protected void updateFilter(Set<String> filterValues) {
         filterDialog.openDialog();
     }
 
+    @Override
+    protected boolean shouldUpdatedItemRemainInList(FeedItem item) {
+        SharedPreferences prefs = getActivity().getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE);
+        FeedItemFilter feedItemFilter = new FeedItemFilter(prefs.getString(PREF_FILTER, ""));
+
+        if (feedItemFilter.isShowDownloaded()) {
```

Comment:
Items can be updated for other reasons, too. This line is missing a check if the item is actually no longer downloaded.